analyzers/src/SonarAnalyzer.CSharp/Rules/Hotspots/CookieShouldBeHttpOnly.cs

```diff
@@ -50,12 +54,40 @@ public sealed class CookieShouldBeHttpOnly : ObjectShouldBeInitializedCorrectlyB
 
         internal CookieShouldBeHttpOnly(IAnalyzerConfiguration analyzerConfiguration) : base(analyzerConfiguration, DiagnosticId, MessageFormat) { }
 
+        protected override Func<bool> DetermineIfDefaultConstructorIsSafe(SonarAnalysisContext context, AnalyzerOptions options)
```

Comment:
This should do the trick as well
```suggestion
        protected override bool DefaultConstructorIsSafe(SonarAnalysisContext context, AnalyzerOptions options)
```

Reply:
I also wanted to simplify the method name a little with this

Reply:
The tracker have `isDefaultConstructorSafe` argument. Let's unify the wording to `IsDefaultConstructorSafe` here.